packages/core/datadog-configuration.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+def getConfigurationFilePath() {
+    if (project.hasProperty("datadog")) {
+        if (project.datadog.configurationFilePath) {
+            return project.datadog.configurationFilePath
+        }
+    }
+    return '../../datadog-configuration.json'
+}
+
+def getBuildAssetsDirectory(sourceName) {
+    return project.layout.buildDirectory
+        .dir("generated/assets/datadog/configuration$sourceName")
+}
+
+def getConfigurationFile() {
+    def filePath = getConfigurationFilePath()
+    def rootProjectPath = projectDir.getAbsolutePath()
+
+    // The path to the configuration file is relative to the app's android/app/ directory.
+    def absoluteFilePath = "$rootProjectPath/$filePath"
+
+    assert file(absoluteFilePath).exists() : "Datadog configuration file at $absoluteFilePath does not exist"
```

Comment:
do we want to crash user build here? or simply make a warning and continue (but in this case I guess SDK won't be initialized properly unless there is a code to initialize it on RN side?)

Reply:
I think we can crash the user build. If they added the Gradle script but we can't pick up the file, we should let them know clearly.
That will also make it easier for support, having a misplaced file can have multiple symptoms depending on the configuration.